courses/spark_for_ada_programmers/7_specification_language.rst

```diff
@@ -70,26 +70,26 @@ Simple Expressions
 Richer Expressions
 --------------------
 
-* Counterparts of conditional statements
+* Counterparts of **conditional** statements
 
   - *if expressions* are the counterpart of *if statements*
   - *case expressions* are the counterpart of *case statements*
 
-* Expressions over a collection (range or array or...)
+* Expressions over a **collection** (range or array or...)
 
-  - *universally quantified expression* for properties over all elements
```

Comment:
I thought this needed to be aligned with "Expressions" above to work as sub-bullet points. no?

Reply:
it does not but that's an oversight, good catch !